src/request/src/response.js

```diff
@@ -150,18 +183,20 @@ export class KinveyResponse extends Response {
         || name === 'BlobNotFound'
         || name === 'DocumentNotFound') {
       error = new NotFoundError(message, debug, code, kinveyRequestId);
-    } else if (name === 'ParameterValueOutOfRangeError') {
+    } else if (name === 'ParameterValueOutOfRange') {
       error = new ParameterValueOutOfRangeError(message, debug, code, kinveyRequestId);
     } else if (name === 'ServerError') {
       error = new ServerError(message, debug, code, kinveyRequestId);
+    } else if (name === 'StaleRequest') {
+      error = new StaleRequestError(message, debug, code, kinveyRequestId);
+    } else if (name === 'UserAlreadyExists') {
+      error = new UserAlreadyExistsError(message, debug, code, kinveyRequestId);
+    } else if (name === 'WritesToCollectionDisallowed') {
+      error = new WritesToCollectionDisallowedError(message, debug, code, kinveyRequestId);
     } else {
       return super.error;
     }
 
-    // if (isDefined(name)) {
```

Comment:
I believe the developer who reported the ticket is looking for the opposite i.e. `InvalidCredentialsError`. It's possible that either our docs are incorrect or we used `InvalidCredentialsError` as the name in a previous release. In any case, we should make sure it's clear to them what to expect.